apps/elastic/src/app/install/actions.tsx

```diff
@@ -0,0 +1,87 @@
+'use server';
+import { logger } from '@elba-security/logger';
+import { z } from 'zod';
+import { getRedirectUrl } from '@elba-security/sdk';
+import { RedirectType, redirect } from 'next/navigation';
+import { isRedirectError } from 'next/dist/client/components/redirect';
+import { ElasticError } from '@/connectors/elastic/common/error';
+import { env } from '@/env';
+import { registerOrganisation } from './service';
+
+const formSchema = z.object({
+  organisationId: z.string().uuid(),
+  apiKey: z.string().min(1),
```

Comment:
```suggestion
  apiKey: z.string().min(1, {
    message: 'API Key is required',
  }),
```
else this would be the error message `String must contain at least 1 character(s)`